src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -20,18 +20,25 @@
 /** The intake of the robot. */
 public class Intake extends SubsystemBase {
 
-  private CANSparkMax intake;
+  private CANSparkMax intakeLower;
+  private CANSparkMax intakeUpper;
   private DigitalInput limitSwitch;
 
   /** Creates a new Intake. */
   public Intake() {
-    this.intake = new CANSparkMax(Constants.Intake.INTAKE_MOTOR_ID, MotorType.kBrushless);
+    this.intakeLower =
+        new CANSparkMax(Constants.Intake.LOWER_INTAKE_MOTOR_ID, MotorType.kBrushless);
+    this.intakeUpper =
+        new CANSparkMax(Constants.Intake.UPPER_INTAKE_MOTOR_ID, MotorType.kBrushless);
     this.limitSwitch = new DigitalInput(Constants.Intake.INTAKE_LIMIT_SWITCH_DIO_PORT);
+
+    intakeLower.follow(intakeUpper, Constants.Intake.INTAKE_FOLLOWER_INVERSE_STATE);
+    intakeUpper.setInverted(Constants.Intake.UPPER_INTAKE_MOTOR_INVERTED);
```

Comment:
Having these in this order won't cause `intakeLower` to unfollow, will it?

Reply:
The docs for rev state "Only voltage output is mirrored. Settings changed on the leader do not affect the follower." so i dont see why it would change anything